HPHBhelper/hogwarts/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+# Create your models here.
+class Games(models.Model):
+    shortcode = models.CharField(max_length=4)
+    isActive = models.BooleanField(default=False)
+    host = models.ForeignKey(Players)
+
+    class TurnOrder(models.Model):
+        #The turn order is set after a game is created but before it starts
+        pass
+
+
+class Players(models.Model):
+    name = models.CharField(max_length=20)
+    game = models.ForeignKey(Games)
+
+    class Characters(models.IntegerChoices):
+        HARRY_POTTER = 1
+        RON_WEASLEY = 2
+        HERMIONE_GRANGER = 3
+        GINNY_WEASLEY = 4
+        LUNA_LOVEGOOD = 5
+
+    player_character = models.IntegerField(choices=Characters.choices)
+    health = models.IntegerField(default=10)
+    coins = models.IntegerField(default=0)
+    damage_tokens = models.IntegerField(default=0)
+
+class Villians(models.Model):
```

Comment:
is villain a sort of player? if so you might use inheritance here

for example both the Players and Villains have a name attribute and a bunch of metrics IntegerFields, you could abstract that in a parent class (but leaving it like this is fine too for now)

Reply:
I hadn't thought of it that way, I like it! I'll revisit that when adding the villians in, for now I'm removing it. The amount of work a player will have to do for that during game setup isn't worth it without more features.